llmfoundry/models/hf/hf_base.py

```diff
@@ -181,6 +192,17 @@ def build_config(
             False,  # Necessary due to https://github.com/huggingface/transformers/issues/28056
         )
 
+        if cls.subselect_config_attr is not None and hasattr(
+            config,
+            cls.subselect_config_attr,
+        ):
+            config = getattr(config, cls.subselect_config_attr)
```

Comment:
Could you add a nice error for when the user inputs a subselect_config_attr that is not found?